src/app/containers/FrontPageMain/index.jsx

```diff
@@ -1,16 +1,30 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useContext } from 'react';
 import { shape } from 'prop-types';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import { frontPageDataPropTypes } from '../../models/propTypes/frontPage';
 import { GhostWrapper, GridItemConstrainedLarge } from '../../lib/styledGrid';
+import { ServiceContext } from '../../contexts/ServiceContext';
 import FrontPageSection from '../FrontPageSection';
 import deepGet from '../../helpers/json/deepGet';
 
 const FrontPageMain = ({ frontPageData }) => {
+  const { product, serviceLocalizedName, translations } = useContext(
+    ServiceContext,
+  );
+  const { home } = translations;
+
   const groups = deepGet(['content', 'groups'], frontPageData);
 
+  const offScreenText = (
+    <Fragment>
+      <span lang="en-GB">{product}</span>, {serviceLocalizedName} - {home}
```

Comment:
News don't have `serviceLocalizedName` defined . How do we intend to handle that?

Reply:
Yeah, but we won't be using the FrontPage for `news` right?

Reply:
I don't know what will be used for `news`.